tests/build/depfile_numpy.srctree

```diff
@@ -0,0 +1,52 @@
+# tag: numpy
+
+PYTHON -m Cython.Build.Cythonize -M dep_np.pyx
+PYTHON check_np.py
+
+######## dep_np.pyx ########
+
+cimport numpy as np
+np.import_array()
+
+
+
+######## check_np.py ########
+
+import os
+import re
+
+import numpy as np
+import Cython
+
+with open("dep_np.c.dep", "r") as f:
+    contents = f.read().replace("\n", " ").replace("\\", "")
+
+contents = contents.split()
+
+cy_prefix, _ = os.path.split(Cython.__file__)
+contents = [fname.replace(cy_prefix, "cy_prefix") for fname in contents]
+
+np_prefix, _ = os.path.split(np.__file__)
+contents = [fname.replace(np_prefix, "np_prefix") for fname in contents]
+
+# filter out the version number from `np_prefix/__init__.cython-30.pxd`.
+contents = [re.sub('[.]cython-[0-9]+', '', entry) for entry in contents]
```

Comment:
The `cython-30.pxd` was a Cython 3 invention. 0.29.x just ignores these files and finds the plain .pxd.

(I'm not sure if this is important here for the sake of the tests)

Reply:
Do you want me to remove it? 
So far I exercised a TDD to the letter --- stopped the editor as soon as tests passed :-), to minimize the difference between patches.

Reply:
Probably? I don't think these files should appear in the 0.29.x list of dependencies, so filtering them out is hiding a bug

Reply:
OK, will do, after the CI gives the full list of things it does not like. One other question is the codestyle CI failure --- this does not seem related, is it known?

Reply:
> The `cython-30.pxd` was a Cython 3 invention. 0.29.x just ignores these files and finds the plain .pxd.

In fact, that 0.29.x ignores it is the whole point that this file suffix exists at all. :)